src/test/java/com/mongodb/jdbc/MongoSQLResultSetMetaDataTest.java

```diff
@@ -37,17 +37,17 @@ void initMocks() {
 
     @Test
     void testGetColumnCount() throws SQLException {
-        assertEquals(9, MongoSQLResultSetMetaDataTest.resultSetMetaData.getColumnCount());
+        assertEquals(10, MongoSQLResultSetMetaDataTest.resultSetMetaData.getColumnCount());
```

Comment:
Now that I added the "all types collection", should I rollback these initial test updates that simply added a document to the previous mock collection? Technically this change is now outside the scope of this ticket and doesn't add any new coverage for work done _in this ticket_, but does add some additional coverage we didn't previously have in the unit tests. Since it is already written, I'd gladly leave it. But I can also take it away very easily.

Reply:
Yea it looks like the initial tests for result set included only types that didn't map to types.OTHER, but that's just speculation.